### PR TITLE
Feature/storage scripts

### PR DESCRIPTION
**Pull request by GustavoRSSilva**
New script to add a advertisement address to the advertisement address

### CODE REVIEW

File: scripts/advertisement-storage.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
+const AdvertisementStorage = artifacts.require("./AdvertisementStorage.sol");
```
Comment (DioPires) [inline scripts/advertisement-storage.js:1]: #### Tests:
```diff 
+ 28 passing
```
#### Test coverage: 
80.25 %

Comment (DioPires) [inline scripts/advertisement-storage.js:1]: #### Tests:
```diff 
+ 28 passing
```
#### Test coverage: 
80.25 %

Comment (DioPires) [inline scripts/advertisement-storage.js:1]: #### Tests:
```diff 
+ 28 passing
```
#### Test coverage: 
80.25 %

Comment (DioPires) [inline scripts/advertisement-storage.js:1]: #### Tests:
```diff 
+ 28 passing
```
#### Test coverage: 
80.25 %

**Comment by GustavoRSSilva** [comment]
added the documentation:

https://github.com/AppStoreFoundation/asf-contracts/wiki/AdvertisementStorage-contract

Added the advertisement contract to the AdvertisementStorage in the ropsten network

screenshot from 2018-06-05 11-19-06

File: scripts/advertisement-storage.js (hunk 1 above)
Comment (DioPires) [inline scripts/advertisement-storage.js:1]: #### Tests:
```diff 
+ 28 passing
```
#### Test coverage: 
80.25 %

Comment (DioPires) [inline scripts/advertisement-storage.js:1]: #### Tests:
```diff 
+ 28 passing
```
#### Test coverage: 
80.25 %

Comment (DioPires) [inline scripts/advertisement-storage.js:1]: #### Tests:
```diff 
+ 28 passing
```
#### Test coverage: 
80.25 %

**Review by tcrosado — APPROVED** [review]

**Review by DioPires — APPROVED** [review]

**Pull request merged**